.travis.yml

```diff
@@ -4,20 +4,14 @@ language: cpp
 cache: ccache
 
 ## Enable sudo
-sudo: required
+sudo: false
```

Comment:
Whot? Why isn't sudo needed when using docker compose? According to https://docs.travis-ci.com/user/docker/ you need `sudo: required` to run docker.

Reply:
Have you seen the link I posted? :)

I think this is it:

> Travis CI automatically routes builds to run on Trusty sudo: required when services: docker is configured. We do not currently support use of Docker on OS X.

I wonder why just before that note they say you should use `sudo: required` if in the next line they say is automatically done when `services: - docker` is used :thinking:

Reply:
Yup, it looks like it's a sudo: https://travis-ci.org/sociomantic-tsunami/libdrizzle-redux/jobs/276921313#L125